assets/package.json.default

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "miranj/craft-build",
+    "version": "0.0.1",
+    "main": "gulp.js",
+    "author": "Miranj",
+    "devDependencies": {
+        "autoprefixer": "^9.4.7",
+        "fontfaceobserver": "^2.1.0",
+        "fontloader": "gist:4e2c2cba3affe643ff18bd4eaf869678",
```

Comment:
Nice! Did not know that npm can pull in from gists as well.

Reply:
I had to fork your gist to add a package.json to enable that.